Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -81,14 +85,16 @@
       - id: CigarGoldCase
         prob: 0.25
       - id: ClothingBeltSheathFilled
+        prob: 0.25
 #      - id: DoorRemoteCommand
       - id: ClothingNeckCloakCapFormal
       - id: ClothingUniformJumpsuitCapFormal
       - id: ClothingUniformJumpskirtCapFormalDress
       - id: RubberStampCaptain
       - id: WeaponAntiqueLaser
+        prob: 0.25
       - id: JetpackCaptainFilled
-      - id: MedalCase
+#      - id: MedalCase
       - id: ClothingHeadHatCapcap
 
```

Comment:
we dont (or shouldnt) have regular captain lockers mapped anywhere, we strictly use an empty one filled by the mapper, so these changes are probably moot

Reply:
Probably but it will allow using it as it is.

Reply:
Made it a bit more empty by removing guns and sword by default.